xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd

```diff
@@ -406,16 +406,19 @@ Rail transport, Roads and Road transport
 	</xsd:group>
 	<xsd:group name="LinePropertiesGroup">
 		<xsd:annotation>
-			<xsd:documentation>Property Elements of a LINE.</xsd:documentation>
+			<xsd:documentation>Property elements of a LINE.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:choice minOccurs="0">
+			<xsd:element ref="AuthorityRef" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Primary OPERATOR of LINE.</xsd:documentation>
+					<xsd:documentation>A line should always have an Authority, even when the information is provided in a ResponsibilitySet.</xsd:documentation>
```

Comment:
Do you want to write AUTHORITY here or isn't that a Transmodel concept?

Reply:
changed